wire/msgmixpairreq.go

```diff
@@ -0,0 +1,426 @@
+// Copyright (c) 2023 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package wire
+
+import (
+	"bytes"
+	"fmt"
+	"hash"
+	"io"
+
+	"github.com/decred/dcrd/chaincfg/chainhash"
+)
+
+const (
+	// MaxMixPairReqScriptClassLen is the maximum length allowable for a
+	// MsgMixPairReq script class.
+	MaxMixPairReqScriptClassLen = 32
+
+	// MaxMixPairReqUTXOs is the maximum number of unspent transaction
+	// outputs that may be contributed in a single mixpairreq message.
+	MaxMixPairReqUTXOs = 512 // XXX: PNOOMA
+
+	// MaxMixPairReqUTXOScriptLen is the maximum length allowed for the
+	// unhashed P2SH script of a UTXO ownership proof.
+	// XXX: might want to limit this to standard script sizes
```

Comment:
Reminder.  Need to make a decision here.  Personally, I don't think we should limit to the standard sizes because this value will effectively become part of consensus and standardness limits are explicitly not part of consensus.

Reply:
Removed this comment and left the value the same (to match txscript.MaxScriptSize).